src/App.js

```diff
@@ -28,22 +28,17 @@ const alex = {
 
 const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
+    <Person
+      person={misha}
+    />
```

Comment:
```suggestion
    <Person person={misha} />
```

Reply:
Check point 2 in checklist
https://github.com/mate-academy/react_sum/blob/master/checklist.md

Reply:
Fixed